src/msg_handler.rs

```diff
@@ -43,6 +43,7 @@ impl MessageHandler for DefaultMessageHandler {
             msg::Any::LoadModule => ops::modules::op_load_module,
             msg::Any::ImageApplyTransforms => ops::image::op_image_transform,
             msg::Any::AcmeGetChallenge => ops::acme::op_get_challenge,
+            msg::Any::OsExit => ops::os::op_exit,
```

Comment:
I think we'll want a separate message handler that allows for this message. Maybe this new one embeds the default message handler and passes messages along.